modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/ApiClientHolder.mustache

```diff
@@ -0,0 +1,9 @@
+{{>licenseInfo}}
+
+package {{invokerPackage}};
+
+public interface ApiClientHolder {
```

Comment:
This doesn't seem to be used anywhere. What is it for ?

Reply:
Just to elaborate a bit more on this - the client Api's generated by swagger do not implement any interface. ApiClientHolder is what it describes - a holder of ApiClient. It acts for us both as a way to reach ApiClient within generated Api as well as a way to hold more generated Api's and organize them meaningfully (better than using plain Object).

If I think from the top of my head, the other possibility would be to introduce something like SwaggerClient or SwaggerApi which would characterize the generated Api and allow manipulations with it. It would expose ApiClient within any such Api - just like ApiClientHolder does now.

What do you think?